src/Package/Impl/DataInspect/DataSource/GridDataSource.cs

```diff
@@ -31,5 +30,10 @@ internal sealed class GridDataSource {
                 }
             }
         }
+
+        public static Task<IGridData<string>> GetGridDataAsync(string expression, GridRange? gridRange, ISortOrder sortOrder = null) =>
+            GetGridDataAsync(
+                VsAppShell.Current.ExportProvider.GetExportedValue<IRSessionProvider>().GetInteractiveWindowRSession(),
```

Comment:
This is the way that it was accessed originally before I split this up. There's really a lot to refactor here - e.g. this class shouldn't even have this overload at all, the caller should just supply session - but for now I'd just like to fix the issues / failing tests.